config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,16 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :access_token,
+  :code,
+  :email,
```

Comment:
is this really sensitive? just curious if it's not going to make it difficult to debug things and problems...

Reply:
i was thinking more in GDPR terms i guess. the trello card did mention it as being sensitive hence the inclusion